docker-compose-dev.yml

```diff
@@ -6,7 +6,7 @@ services:
       dockerfile: ./Dockerfile
       args:
         MIX_ENV: dev
-    image: aeternity/ae_mdw_dev:master
+    image: aeternity/ae_mdw:latest
```

Comment:
I think that this should be master, so you can either pull the latest master or build it

Reply:
Good idea. Done